sdks/python/apache_beam/ml/rag/chunking/langchain_test.py

```diff
@@ -167,7 +167,7 @@ def test_huggingface_tokenizer_splitter(self):
       def check_token_lengths(chunks):
         for chunk in chunks:
           # Verify each chunk's token length is within limits
-          num_tokens = len(tokenizer.encode(chunk.content.text))
+          num_tokens = len(tokenizer.tokenize(chunk.content.text))
```

Comment:
Apparently, this is a known [issue](https://github.com/langchain-ai/langchain/issues/30184) where the `encode` function adds 2 extra tokens. Replaced it with `tokenize` since it solved the problem [here](https://github.com/langchain-ai/langchain/pull/30185/files).